src/operations/identify.rs

```diff
@@ -1,18 +1,33 @@
 use crate::{error::MagickError, image::Image};
 
+// https://imagemagick.org/script/command-line-options.php#identify
 pub fn identify(image: &mut Image) -> Result<(), MagickError> {
     println!("{}", identify_impl(image));
     Ok(())
 }
 
 fn identify_impl(image: &Image) -> String {
-    format!("{}x{}", image.pixels.width(), image.pixels.height())
+    let parts: Vec<String> = vec![
+        image.properties.filename.to_str().map(str::to_owned),
+        image.format.map(|f| f.extensions_str()[0].to_uppercase()),
+        Some(format!(
+            "{}x{}",
+            image.pixels.width(),
+            image.pixels.height()
+        )),
```

Comment:
Could you put each of these into a descriptive variable? Say, `let dimensions = Some(format!(...` and so on? That would be much more readable.